.github/workflows/cypress-workflow.yml

```diff
@@ -9,6 +9,8 @@ jobs:
     name: Run Cypress E2E tests
     runs-on: ubuntu-latest
     env:
+      # allow unsecure commands to perform further set environment variable commands
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
I'm assuming this for the [recent set-env and add-path deprecation changes](https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/).

Are the errors coming from actions that we're depending on that haven't updated to use the new model or environment variables that we are settings ourselves? If it's from dependent actions, we should double check to see if we can use some newer version if they've made the change to environment files. If it's from the way we've written the workflow, we should [move to using environment files](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#environment-files).

If any actions haven't updated to use environment files yet then this seems fine for now. Just want to make sure we're following best practices if the option is available.

Reply:
Removed this and instead upgrade `cypress-io/github-action` from @v1 to@v2.5.0`